x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/service/ElasticServiceAccounts.java

```diff
@@ -29,7 +29,14 @@
             new String[] { "monitor", "manage_own_api_key" },
             new RoleDescriptor.IndicesPrivileges[] {
                 RoleDescriptor.IndicesPrivileges.builder()
-                    .indices("logs-*", "metrics-*", "traces-*", "synthetics-*", ".logs-endpoint.diagnostic.collection-*")
+                    .indices(
+                        "logs-*",
+                        "metrics-*",
+                        "traces-*",
+                        "synthetics-*",
+                        ".logs-endpoint.diagnostic.collection-*",
+                        ".logs-endpoint.action.responses-*"
```

Comment:
Can you also add this new index pattern to the existing test in `ElasticServiceAccountsTests`. Basically add the new name under the following line:

https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/service/ElasticServiceAccountsTests.java#L166

Reply:
added here: https://github.com/elastic/elasticsearch/pull/80231/commits/7363f755a2a4d5b53ba0372113dd0469b58c5570